### PR TITLE
Partial translation of ch01-03-how-rust-is-made-and-nightly-rust.md

### PR DESCRIPTION
**Pull request by Osovich**
From line 1 to line 8, around 85 words. Work in progress.

### CODE REVIEW

**Comment by thecodix** [comment]
Hey, will you add some more to this PR? if you are working on it, please state if it's a work on progress :)

**Comment by Osovich** [comment]
oh, yes, sorry, forgot to state that, I will continue on it!

**Comment by Osovich** [comment]
Translated some more, I'll keep working on this file later, you can check it and if you like it you can merge the pull request. Thanks!

File: second-edition/src/ch01-03-how-rust-is-made-and-nightly-rust.md [hunk 1]
```diff
@@ -1,28 +1,30 @@
...
-is the way we can change and improve Rust while making sure that using Rust
-stays nice, stable, and boring.
-
-Our guiding principle for Rust releases is this: you should never have to fear
-upgrading to a new version of stable Rust. Each upgrade should be painless. At
-the same time, the upgrade should bring you new features, fewer bugs, and
-faster compile times.
+## Cómo está Hecho Rust y "Rust Nocturno"
+
+Antes de cavar en el lenguaje, quisiéramos terminar el capítulo introductorio
```
Comment (thecodix) [inline second-edition/src/ch01-03-how-rust-is-made-and-nightly-rust.md]: 'cavar' -> 'indagar'

**Comment by thecodix** [comment]
Then I will wait until you make the rest of commits and keep this PR open.

Bear in mind this is an important chapter to understand Rust idiosyncrasy, you have a big responsibility here :)

File: second-edition/src/ch01-03-how-rust-is-made-and-nightly-rust.md (hunk 1 above)
Comment (Osovich) [reply second-edition/src/ch01-03-how-rust-is-made-and-nightly-rust.md]: sorry!

**Comment by Osovich** [comment]
Still in progress, I'm just saving progress if anything bad happens!

**Comment by Osovich** [comment]
Finished translation, let me know if there's any correction that needs to happen!

File: second-edition/src/ch01-03-how-rust-is-made-and-nightly-rust.md [hunk 2]
```diff
@@ -164,63 +176,57 @@ beta-x86_64-pc-windows-msvc
...
+
+¡Ya está listo para un buen comienzo en su viaje con Rust! En este capítulo usted ha:
+
+* Aprendido qué hace a Rust único
+* Instalado las últimas versiones estables de Rust
+* Escrito un programa "Hello, world!" usando `rustc` directamente y usando las
+convenciones de `cargo`
+* Encontrado sobre el cómo Rust está desarrollado
+
+Este es un buen momento para escribir un programa substancioso, para acostumbrarse
```
Comment (thecodix) [inline second-edition/src/ch01-03-how-rust-is-made-and-nightly-rust.md]: 'substancioso' -> 'relevante' or 'importante'

File: second-edition/src/ch01-03-how-rust-is-made-and-nightly-rust.md [hunk 3]
```diff
@@ -164,63 +176,57 @@ beta-x86_64-pc-windows-msvc
...
-program. If you’d rather start by learning about how common programming
-concepts work in Rust, see Chapter 3.
+Ahora, cada vez que llame `rustc` o `cargo` dentro de *~/projects/needs-nightly*,
+`rustup` se encargará de que usted esté usando la versión nocturna de Rust, en
+vez de su versión estable por defecto de Rust. ¡Esto es útil cuando quiere tener
+muchos proyectos de Rust!
+
+### El Proceso RFC y Los Equipos
+
+Así que ¿Cómo aprende usted de estas nuevas funcionalidades? El modelo dedesarrollo
```
Comment (thecodix) [inline second-edition/src/ch01-03-how-rust-is-made-and-nightly-rust.md]: de desarrollo

File: second-edition/src/ch01-03-how-rust-is-made-and-nightly-rust.md (hunk 2 above)
Comment (Osovich) [reply second-edition/src/ch01-03-how-rust-is-made-and-nightly-rust.md]: On it.

File: second-edition/src/ch01-03-how-rust-is-made-and-nightly-rust.md (hunk 3 above)
Comment (Osovich) [reply second-edition/src/ch01-03-how-rust-is-made-and-nightly-rust.md]: My bad, sorry

**Comment by Osovich** [comment]
I'm here for anything that still needs fixing!

**Comment by thecodix** [comment]
Merged, thanks!

**Pull request merged**